templates/contrib/opening_hours/opening-hours-widget.html.twig

```diff
@@ -0,0 +1,80 @@
+{#
+/**
+ * @file
+ * Template for the Opening Hours : OpenNow formatter.
+ *
+ * Available variables:
+ * - preview_widget : Contains the preview widget.
+ * - widgets : The widgets to show.
+ * - service_id : The opening hours Service ID.
+ * - channel_id : The opening hours Channel ID.
+ * - date : The date to show the widget for.
+ *
+ * @see template_preprocess_vg_social_link()
+ *
+ * @ingroup themeable
+ */
+#}
+
+{{ attach_library('gent_base/opening_hours') }}
+
+<div class="openinghours-wrapper">
+  {% if display_title %}
+    <h3 class="openinghours-channel-title" data-service="{{ service_id }}" data-channel="{{ channel_id }}"></h3>
```

Comment:
Are we sure service_id and channel_id are never empty here?

Reply:
This is a copy of the opening_hours template with some added markup, so no: it won't generate errors. These variables are always defined (even if they have value `NULL`)